### DISCUSSION TITLE
Visual Editor PrimeNG didn't work

### DISCUSSION THREAD

**Discussion by MarielaRH** [primefaces/community#1196] [PrimeNG] [answered]
I'm trying to configure a new custom theme in primeNG but the page didn't work, are there another option to make a custom theme? 
![Captura de pantalla 2024-02-14 144545](https://github.com/primefaces/community/assets/68169069/566b84c5-40a9-4cef-9e87-88fa7e57cfea)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

A new and enhanced version of visual editor is in development progress. We'll replace the outdated one with a new and modern one.

**Comment by autonomobil** [comment]
Any ETA for that?
